src/drive/web/modules/drive/Toolbar/components/Scanner.jsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react'
+
+import { withClient } from 'cozy-client'
+
+import { CozyFile } from 'cozy-doctypes'
+
+class Scanner extends React.Component {
+  state = {
+    status: 'iddle',
```

Comment:
```suggestion
    status: 'idle',
```

Reply:
And maybe we can expose constants for the different statuses and conflict strategies? Avoids hard to detect typos.